Xamarin.Forms.Core/Items/ItemsView.cs

```diff
@@ -97,4 +98,29 @@ protected virtual void OnScrollToRequested(ScrollToRequestEventArgs e)
 			ScrollToRequested?.Invoke(this, e);
 		}
 	}
+
+	public enum SelectionMode 
+	{
+		None,
+		Single,
+		Multiple
+	}
+
+	public class SelectionChangedEventArgs : EventArgs
+	{
+		public IReadOnlyList<object> PreviousSelection { get; }
+		public IReadOnlyList<object> CurrentSelection { get; }
+
+		public SelectionChangedEventArgs(object previousSelection, object currentSelection)
+		{
+			PreviousSelection = previousSelection != null ? new List<object> { previousSelection } : new List<object>();
```

Comment:
- the list will never grows, construct as `new List<object> (1) { previousSelection}`
- if the empty case happens often, you might want to have an static `EmptyList` that you can reuse

Reply:
Done and done.